raiden/messages.py

```diff
@@ -1830,7 +1832,7 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'type': self.__class__.__name__,
             'chain_id': self.chain_id,
             'nonce': self.nonce,
-            'token_network_address': to_normalized_address(self.token_network_address),
+            'token_network_address': to_checksum_address(self.token_network_address),
```

Comment:
hmm this is not looking good. Every other message we send around exports to the `normalized_address`. Why should this message be different? I would instead adjust the PFS and not the client here.

Reply:
Is there a reason not to use checksum addresses? Consistency is nice, but maybe there isn't a conscious thought behind using normalized aka lower case hex addresses.

Reply:
Can't say for sure. But the only reason for this kind of checksumming is to protect the user from mistyping. So it serves no purpose in the wire between raiden clients.

Reply:
A checksum might always be useful, but I don't really care. Changed it.